packages/ui/src/accounts/hooks/useBalance.ts

```diff
@@ -11,16 +9,15 @@ import { Balances } from '../types'
 import { useMyBalances } from './useMyBalances'
 
 export const useBalance = (address: Address = ''): Balances | null => {
-  const { api, connectionState } = useApi()
+  const { api } = useApi()
 
   const allMyBalances = useMyBalances()
   const myBalances = allMyBalances?.[address]
 
-  const balancesObs = useMemo(
+  const balances = useObservable(
```

Comment:
I think this one could use `useFirstObservableValue` or am I missing something ?

Reply:
Yeah, you are right. I forgot that balances from wallet addresses are coming from BalancesContext which is using `useObservable`, therefor it will get every balance update (if someone transfers us some money). So in here there is no reason to observe it constantly as well, nice catch! :)